src/test/java/org/fcrepo/importexport/integration/RoundtripIT.java

```diff
@@ -462,9 +466,81 @@ public void testRoundtripExternal() throws Exception {
         final Model model = getAsModel(file1desc);
         assertTrue(model.contains(binary, RDF_TYPE, createResource(LDP_NON_RDF_SOURCE)));
         assertTrue(model.contains(binary, createProperty(SKOS_PREFLABEL), "original version"));
-        assertTrue(model.contains(binary, createProperty(PREMIS_SIZE), longLiteral("0")));
-        assertTrue(model.contains(binary, createProperty(PREMIS_DIGEST),
-                createResource("urn:sha1:da39a3ee5e6b4b0d3255bfef95601890afd80709")));
+        assertTrue(model.contains(binary, createProperty(PREMIS_SIZE), longLiteral("-1")));
+    }
+
+    @Test
+    public void testRoundtripExternalProxy() throws Exception {
+        final UUID uuid = UUID.randomUUID();
+        final String baseURI = serverAddress + uuid;
+        final URI res1 = URI.create(baseURI);
```

Comment:
Do we need to create res1/container in this test? The binary isn't being stored in it, it just appears to be a sibling as far as I can tell. For the sake of slightly simplifying this test (roundtripping tests are a bit of a beast), perhaps we should remove them?

Reply:
good idea.

Reply:
@bbpennel : Good idea.  Removed.